cfgov/unprocessed/css/organisms/info-unit-group.less

```diff
@@ -63,8 +63,33 @@
         // Adds margin top to info units that follow another
         margin-top: unit( @grid_gutter-width / @base-font-size-px, em ) !important;
     }
+    
+    .content-l__top-divider + .content-l__top-divider {
+        // Adds top divider between info units
+        .grid_column__top-divider;
+        margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
+    }
+    
+    .respond-to-max( @bp-xs-max, {
+        // Adds top divider between info unit cols on small screens
+        .content-l__top-divider {
+            .content-l_col + .content-l_col {
+                .grid_column__top-divider;
+                margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
+            }
+        }
+    });
 
     .respond-to-min( @bp-sm-min, {
+        .content-l__top-divider + .content-l__top-divider {
+            // Matches top divider width to cols on larger screens
+            &:before {
+                margin-left: 15px !important;
```

Comment:
Add note as to why `!important` is necessary?

Reply:
Note added in https://github.com/cfpb/cfgov-refresh/pull/3476/commits/cd56484fa764a5930cc8390511c7c89ca8bb1889.